copy-from-usb/src/main/java/com/playground/Main.java

```diff
@@ -2,15 +2,58 @@
 
 import com.playground.utils.FileUtils;
 
-import java.io.File;
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
 
 public class Main {
 
-    public static void main(String[] args) {
-        System.out.println("main()");
-        for (File file: FileUtils.getNonLocalDiskPath()) {
+    private static final List<String> EXTENSITIONS = new ArrayList<>();
+
+    static {
+        EXTENSITIONS.add("txt");
+        EXTENSITIONS.add("png");
+        EXTENSITIONS.add("jpg");
+    }
```

Comment:
`private static final List<String> EXTENSITIONS = List.of("txt", "png", "jpg");`